### ISSUE TITLE
Font icons for source code callouts

### ISSUE THREAD

**Issue by antoyo** [asciidoctor/asciidoctor-reveal.js#54]
Hello.
The font icons for source code callouts (`<1>`, for instance) does not exist in the `reveal.js` backend.
Could you add them, please?
Thanks.

**Comment by obilodeau** [comment]
Can you provide a minimal example that would reproduce the problem? Thanks!

**Comment by antoyo** [comment]
Yes.
Here is the code:
```adoc
= Presentation
Me
:backend: revealjs
:icons: font
:source-highlighter: highlightjs

== Callout

[source, rust]
----
fn main() {
    println!("Hello World!"); // <1>
}
----
<1> `println!` is a macro.
```
The result:
![img-2016-08-26-080038](https://cloud.githubusercontent.com/assets/584972/18004660/43c104cc-6b63-11e6-9ae0-0a4565173658.png)
And the expected result is to have icons similar to the HTML5 backend:
![img-2016-08-26-080048](https://cloud.githubusercontent.com/assets/584972/18004662/46a3d2e6-6b63-11e6-80fc-a3f52dd5e6c6.png)

**Comment by obilodeau** [comment]
With reveal.js 3.3.0 I think we are closer (uses highlight.js 9 which I think supports callouts):

![adoc-callout](https://cloud.githubusercontent.com/assets/546325/18030230/c2fde33a-6c7c-11e6-9849-d44823ed1377.png)

However, there might be something needed on our end in order to avoid escaping the HTML (or something like that).

Also tested with coderay, pygments and rouge. Nothing works.

**Comment by mojavelinux** [comment]
It's important to note that the appearance of the callout in built-in HTML backend is provided exclusively by CSS. The screenshot @antoyo posted is exactly what you'd expect if CSS is not helping. The built-in stylesheet in Asciidoctor transforms the `<i>` element into an enclosed number using CSS. There's no special thing that the converter does.

**Comment by obilodeau** [comment]
Good to know. I'll add this to the list of CSS that we need to pull in from asciidoctor.

**Comment by obilodeau** [comment]
Fixed by #224. See also #168. Will be part of our next release.
